tests/system/trace/flask/main.py

```diff
@@ -20,9 +20,9 @@
 import sqlalchemy
 
 from opencensus.common.transports import async_
+from opencensus.ext.flask.flask_middleware import FlaskMiddleware
```

Comment:
What do you want to do about this test in the long run? I don't think we should have a dependency on a contrib package in the core library tests, but since we have more work to do to clean up the system tests I don't know that it's worth extracting the test in this PR.

Reply:
In the long run we might want to move integration test into contrib packages.
Both the documentation and CI need to be refactored. Currently we don't have a separate CI for contrib packages, so I just keep this part for now.